tests/ts-app-template-classic/types/ts-app-template/index.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="@embroider/core/virtual" />
```

Comment:
I don't think `@embroider/virtual` is used in the ts-app-template-classic 🤔 do we need this line?

Reply:
right you are! removed